.devops/soak-tests-pipeline.yml

```
@@ -24,12 +24,24 @@ parameters:
     type: string
     default: "auth/getUserInfo"
     values:
-      - auth/getUserInfo
-      - auth/postToken_clientCredentials_PU
-      - auth/postToken_clientCredentials_SIL
-      - auth/postToken_tokenExchange
-      - auth/registerRevokeClient
-      - mypay/configAndLogin
+      - runAll
+      - runAll_Auth
+
+      - Authn_getUserInfo
```

let's use the runAll_<APP_NAME> as first row for each application or domain tested, in order to have a visual separation when exploring the pipeline setup on AzureDevOps
```suggestion
      
      - runAll_Auth
      - Authn_getUserInfo
```